lib/banner.py

```diff
@@ -8,7 +8,8 @@ def print_version():
 
 
 def print_banner():
-    banner = f"""{Fore.CYAN}                         _
+    orange = "\033[38;5;208m"
+    banner = f"""\033[1;92m                         _
```

Comment:
Please include this values inside the Prowler config.

Reply:
Done